app/routes/activists2.tsx

```diff
@@ -80,7 +80,6 @@ export default function ActivistsPage() {
   const loadedItems = itemsFromRemixData(data.collectionItems, infoMap);
 
   return (
-    
```

Comment:
nit:  try to avoid any unnecessary changes

Reply:
Should I put the newline back?

Reply:
Added back in.